images/rootfs.yml.in

```diff
@@ -46,6 +46,10 @@ services:
     image: MONITOR_TAG
     cgroupsPath: /eve/services/monitor
     oomScoreAdj: -999
+  - name: vector
+    image: VECTOR_TAG
+    cgroupsPath: /eve/services/vector
+    oomScoreAdj: -999
```

Comment:
Doesn't this mean we need a vector.touch file watched by watchdog in case vector is killen by OOM?

Reply:
I think it's okay to just restart vector when it's killed. Triggering the watchdog and restarting the whole device will probably help nobody.
I'll add the logic for it